mathmodel.cpp

```diff
@@ -1,285 +0,0 @@
-#include <iostream>
-#include <algorithm>
-#include <cmath>
-
-const double G = 6.67385 * pow(10, -11); //„‡‚ËÚ‡ˆËÓÌÌ‡ˇ ÔÓÒÚÓˇÌÌ‡ˇ
-
-using namespace std;
-
-double size, //‡ÁÏÂ ÍÓ‡·Îˇ
-maxTorque, //ÏÓÏÂÌÚ ‚‡˘ÂÌËˇ
-maxFuel, //Ï‡ÍÒËÏ‡Î¸Ì˚È ‡ÒıÓ‰ ÚÓÔÎË‚‡
-maxImpulse, //Û‰ÂÎ¸Ì˚È ËÏÛÎ¸Ò
-fuelLevel, //ÛÓ‚ÂÌ¸ Ï‡ÒÒÓ‚Ó„Ó ‡ÒıÓ‰‡ ÚÓÔÎË‚‡
-limOver; //ÔÂ‰ÂÎ¸Ì‡ˇ ÔÂÂ„ÛÁÍ‡
-
-struct vec //ÓËÂÌÚ‡ˆËˇ
-{
-double x, y, z;
-};
-vec position; //ÔÓÎÓÊÂÌËÂ
-
-struct m //Ï‡ÒÒ˚ ÍÓ‡·Îˇ Ë ÚÓÔÎË‚‡
-{
-double ship, fuel;
-};
-
-vec rSpeed; //ÒÍÓÓÒÚ¸ ‚‡˘ÂÌËˇ
-
-vec momentRotation; //ÏÓÏÂÌÚ ‚‡˘ÂÌËˇ ‚ÓÍÛ„ Í‡Ê‰ÓÈ ÓÒË
-
-vec rageSim; //‰Ë‡ÔÁÓÌ ÏÓ‰ÂÎËÓ‚‡ÌËˇ
-
-struct airD//ÒÓÔÓÚË‚ÎÂÌËÂ ‚ÓÁ‰Ûı‡
-{
-	double h, p;
-};
-
-double scalar (vec t)
-{
-    double res = sqrt(pow(t.x, 2) + pow(t.y, 2) + pow(t.z, 2));
-    return res;
-}
-
-double airDens(double H)
-{
```

Comment:
There are a lot of constants. If we use them we should explain each constant or group of constants with comments.